modules/opentelemetry/src/main/scala/natchez/opentelemetry/OpenTelemetry.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2019-2020 by Rob Norris and Contributors
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package natchez
+package opentelemetry
+
+import cats.effect._
+import cats.implicits._
+import io.opentelemetry.api.GlobalOpenTelemetry
+import io.opentelemetry.api.trace.Tracer
+import io.opentelemetry.sdk.{OpenTelemetrySdk, OpenTelemetrySdkBuilder}
+
+import java.net.URI
+
+object OpenTelemetry {
+  private final val instrumentationName = "natchez.opentelemetry"
+
+  // Helper methods to help you construct Otel resources that clean themselves up
+  // We need a name so the failure error can contain something useful
+  def lift[F[_]: Async, T: Shutdownable](name: String, create: F[T]): Resource[F, T] =
+    Resource.make(create) { t =>
+      Sync[F].delay { Shutdownable[T].shutdown(t) }
+        .flatMap(Utils.asyncFromCompletableResultCode(s"$name cleanup", _))
+    }
+
+  def entryPoint[F[_] : Sync](uriPrefix: Option[URI] = None, globallyRegister: Boolean = false)(
+    configure: OpenTelemetrySdkBuilder => Resource[F, OpenTelemetrySdkBuilder]
```

Comment:
Would the signature for configure be more accurate to be `OpenTelemetrySdkBuilder => F[OpenTelemetrySdkBuilder]`?  Is there a configuration that you'd want to control a finalizer on it?